src/flycapture2.pyx

```diff
@@ -642,3 +642,33 @@ cdef class Image:
 
     def get_format(self):
         return self.fmt or self.img.format
+        
+    def get_timestamp(self):
+        with nogil:
+            ts=fc2GetImageTimeStamp( &self.img)
+        return ts
+        
+    def save_as_png(self,filename):
```

Comment:
This should be `save_image()` and the format choice should be exposed as an argument in a suitable way.